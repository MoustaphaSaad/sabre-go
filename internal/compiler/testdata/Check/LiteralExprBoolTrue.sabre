package main

func IsValid() bool {
    return true
}
