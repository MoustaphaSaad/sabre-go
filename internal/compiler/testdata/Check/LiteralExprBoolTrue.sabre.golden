>> 	}
>> 	^ 
Error[internal/compiler/testdata/Check/LiteralExprBoolTrue.sabre:5:1]: expected ';' but found '}'
>> 	}
>> 	 ^
>> 	
>> 	 
Error[internal/compiler/testdata/Check/LiteralExprBoolTrue.sabre:5:2]: expected an expression but found ';(\n)'
>> 	
>> 	^
Error[internal/compiler/testdata/Check/LiteralExprBoolTrue.sabre:6:1]: expected '}' but found 'INVALID'
